docs/contributing.md

```diff
@@ -25,7 +25,7 @@ To contribute to Submariner:
 
 1. Fork this repository
 2. Make your changes in the fork
-3. Test your changes by running `make ci e2e status=keep` which deploys [Submariner in a Kind 3 cluster setup](https://github.com/submariner-io/submariner/blob/master/scripts/kind-e2e/README.md) and runs E2E tests
+3. Test your changes by running `make ci e2e` which deploys [Submariner in a Kind 3 cluster setup](https://github.com/submariner-io/submariner/blob/master/scripts/kind-e2e/README.md) and runs E2E tests
```

Comment:
```suggestion
3. Test your changes by running `make e2e status=keep` which deploys [Submariner in a Kind 3 cluster setup](https://github.com/submariner-io/submariner/blob/master/scripts/kind-e2e/README.md) and runs E2E tests
```

In this case, for developers, I'd instruct them to do the keep, so they can re-test again without re-deploying everything, may be we can add "5. Run `make cleanup` to tear down the test clusters once you're finished.

Reply:
What do you mean? The whole point is to remove `status=` flag.. There's also instructions on how to run cleanup..